README.md

```diff
@@ -47,10 +48,16 @@ func main() {
 	}
 
 	if err := bouncer.Init(); err != nil {
-		log.Fatalf(err.Error())
+		log.Fatal(err)
 	}
 
-	go bouncer.Run()
+	ctx, cancel := context.WithCancel(context.Background())
+	defer cancel()
+
+        go func() {
+            bouncer.Run(ctx)
+            cancel()
```

Comment:
are you sure that the cancel() is required here ? Because it seems that it should be taken care of inside the bouncer.Run(which I believe you do, but if bouncer.Run() is done, then no need to stop it (and there's the defer clause that will cancel it anyway)

Reply:
yes of course. thanks